src/examples/icon-button/icon-button-toggle-state.tsx

```diff
@@ -20,7 +20,7 @@ export class IconButtonExample {
         return [
```

Comment:
The default icon in limel-file is `note`, which is missing in the icon-set used in the documentation. But I don't want to change it, since that would affect the default icon when used in the webclient. I don't quite know what to do about that. In the case of the x-icon on the chip, I've let the chip supply the icon itself, but that doesn't work in the case of the default file icon.